### ISSUE TITLE
MSG+ is being replaced with a new service

### ISSUE THREAD

**Issue by adamef93** [tonywagner/EPlusTV#106]
Me again haha

The new "Gotham Sports App" is combining YES and MSG+ into a single service which is, apparently, planned to launch before the start of the NHL and NBA seasons in October. Looks like existing MSG+ will be sunsetted when this launches, which I imagine means the authentication and stream grabbing code will need to be reworked for the new service

Assuming this launches in time, I'd be eternally grateful if it was possible to get this going for this season 💖

https://www.nytimes.com/athletic/5727667/2024/08/28/yes-msg-network-gotham-sports-app-launch/

https://www.prnewswire.com/news-releases/game-the-digital-joint-venture-between-msg-networks-and-the-yes-network-to-launch-the-gotham-sports-app-prior-to-upcoming-nba-and-nhl-regular-seasons-302233459.html

**Comment by m0ngr31** [comment]
It looks like MSG+ and YES will keep running as well? I'm curious to see how it'll turn out. MSG+ was a pretty awful service to try and reverse engineer (terrible interface), so hopefully Gotham is better.

**Comment by adamef93** [comment]
MSG+ is kinda awful to use too, so I can only imagine lol

Yea, it looks like the plan is to bundle MSG+ and YES into the single Gotham front end, but people can still choose to subscribe to either separately or both. Hopefully this means the back end is left alone and your code will still work while both services are just wrapped in a different front end

**Comment by m0ngr31** [comment]
Looks like it finally launched. If someone wants to lend me some creds I can take a look

**Comment by adamef93** [comment]
Awesome! The same creds I email for MSG will work.

**Comment by m0ngr31** [comment]
@adamef93 can you send me the creds again? The link you sent has expired. If it's possible to send your TVE authentication too, that'd help a bunch

**Comment by adamef93** [comment]
Just resent the creds, but can't share the TVE creds since they aren't mine to share. Sorry :/ I hope the MSG/Gotham creds can still be some help

**Comment by m0ngr31** [comment]
Removed MSG+ and replaced it with Gotham in v4.1.0

**Comment by adamef93** [comment]
Hey, what are the environment variables for logging in with Gotham? I changed `MSGPLUS_USER` and `MSGPLUS_PASS` to `GOTHAM_USER/GOTHAM_PASS` and `GOTHAMSPORTS_USER/GOTHAMSPORTS_PASS` but neither is producing a login prompt in the container logs

I'm assuming this change means you were able to get playback working?

---

Here are the logs from when the container starts up and I try loading a current event. I removed the docker volume where where `/app/config/` is mapped and started fresh

```
> eplustv@4.1.0 start
> ts-node -r tsconfig-paths/register index.tsx
=== E+TV v4.1.0 starting ===
Server started on port 8000
=== Getting events ===
=== Done getting events ===
=== Building the schedule ===
=== Done building the schedule ===
Could not get a playlist for channel #9050. Please make sure there is an event scheduled and you have access to it.
```

---

Also should mention that I forgot to change `MSGPLUS=true` at first, but I tried setting it to `GOTHAM=true` and `GOTHAMSPORTS=true` with no change. I rolled back the image I'm using to 4.0.3 to use with MSG+ for tonight's game.
